pkg/ansible/flags/flag.go

```diff
@@ -59,7 +61,16 @@ func (f *Flags) AddTo(flagSet *pflag.FlagSet, helpTextPrefix ...string) {
 	flagSet.IntVar(&f.MaxWorkers,
 		"max-workers",
 		runtime.NumCPU(),
-		"Maximum number of workers to use. Overridden by environment variable.",
+		strings.Join(append(helpTextPrefix,
```

Comment:
In line 44, I noticed I missed removing the `helpTextPrefix ...string` parameter, which we no longer need due to the fact that helm and ansible operator flags are no long comingled in any SDK subcommand CLIs.

Let's just remove that parameter and stick with the simple string usage text.